include/stream_buffer.h

```diff
@@ -202,9 +202,9 @@ typedef struct StreamBufferDef_t * StreamBufferHandle_t;
  * StreamBufferHandle_t xStreamBuffer;
  * const size_t xTriggerLevel = 1;
  *
- *  xStreamBuffer = xStreamBufferCreateStatic( sizeof( ucBufferStorage ),
+ *  xStreamBuffer = xStreamBufferCreateStatic( sizeof( ucStorageBuffer ) - 1,

```

Comment:
It should be `sizeof( ucStorageBuffer )` as the comment above reads:
```
 * // Used to dimension the array used to hold the streams.  The available space
 * // will actually be one less than this, so 999.
#define STORAGE_SIZE_BYTES 1000
```

Reply:
@aggarg, the -1 is required here. This is the main problem this PR is trying to fix.

See this comment:
```
 * @param pucStreamBufferStorageArea Must point to a uint8_t array that is at
 * least xBufferSizeBytes + 1 big.  This is the array to which streams are
 * copied when they are written to the stream buffer.
```

The `- 1` is ensuring the condition above is met. Please reconsider your comment.

Reply:
Thanks for pointing it. You are right.